### PR TITLE
Update mongodb getting started

### LINKED ISSUE #2916: Suggestions for MongoDB "Getting Started Guide"
I have a few suggestions for the MongoDB "Getting Started Guide"

### Improve the Prisma schema

The Prisma schema that's currently used looks as follows:

```prisma
model Post {
  id       String    @id @default(dbgenerated()) @map("_id") @db.ObjectId
  slug     String    @unique
  title    String
  body     String
  comments Comment[]
  user     User      @relation(fields: [userId], references: [id])
  userId   String    @db.ObjectId
}

// Comments contain a comment string and connect back to the post.
// postId must have @db.ObjectId to match up with Post's id type
model Comment {
  id      String @id @default(dbgenerated()) @map("_id") @db.ObjectId
  post    Post   @relation(fields: [postId], references: [id])
  postId  String @db.ObjectId
  comment String
}

model User {
  id    String  @id @default(dbgenerated()) @map("_id") @db.ObjectId
  email String  @unique
  name  String?
  posts Post[]
}
```

I have two main suggestions to improve this.

#### Remove the `Comment` model

IMO, the `Comment` model doesn't add much value compared to the [canonical `User`/`Post` models](https://www.prisma.io/docs/about/prisma-docs/about-the-docs#the-user-and-post-data-model) we typically use in our schema examples. It doesn't introduce any new concepts because it also only shows a one-to-many relationship which readers can already learn by looking at the `User`<->`Post` relation. So in its current form, it seems to add noise without providing educational value. Note that this would be different if the model would show how to use a 1-1 relation or m-n relation.

#### Add an embedded document

With today's release, Prisma introduces support for MongoDB's embedded documents. Since embedded documents are a fundamental concept in MongoDB, I think it might make sense to include them in the "Getting Started Guide" already. This could for example be an `Address` type that's being added to the `User` model.

### Remove the initial warning box

We currently have a warning on top of the guide that looks as follows:

image

It should probably be updated to remove the note that we don't support introspection on MongoDB. Further, I wonder if we need to  point out that _migrations_ are not supported. Mostly because the _concept_ of migrations doesn't really apply to MongoDB in the first place (unless we're talking about _data migrations_, but these are currently not possible with Prisma Migrate either) and it's questionable what Prisma Migrate support for MongoDB actually would look like. So we can probably get rid of the entire warning box?

### Remove `await prisma.$connect()` ƒrom script

The initial, empty script that will show how to send Prisma Client queries currently uses `prisma.$connect()`. IMO, this doesn't provide any value and according to the ["progressive disclosure of complexity"](https://www.jason.af/progressive-disclosure-of-complexity/) principle should be removed. iirc, we used to have an internal conversation about this already and decided against showing `prisma.$connect()` in any "getting started code".

### LINKED ISSUE #3306: MongoDB: clean up links to Atlas in Get Started docs
In both of our Get Started docs (scratch and add) we have two links to Atlas... we should reconsider having just one link.

Should new link go to [the high-level page Getting Started with Atlas (UI or CLI)](https://www.mongodb.com/docs/atlas/getting-started/)
OR
should it go to [a very specific page about how to use the Atlas CLI to create your cluster](https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-quickstart/)?

[Slack convo](https://prisma-company.slack.com/archives/C5Z9TH6N9/p1655839738552839)

### LINKED ISSUE #3340: Getting started with CockroachDB shows incorrect schema
The schema on [this guide](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/using-prisma-migrate-typescript-cockroachdb#creating-the-database-schema) uses an incorrect `@default` annotation. You can't use `autoincrement()` with Cockroach, [`sequence()`](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#sequence) is needed instead.

Pointed out in [this Slack convo](https://prisma.slack.com/archives/CA491RJH0/p1656442329845639)

### LINKED ISSUE #3769: Quickstart and Getting Started Guides should use `npx tsc init` instead of manually creating `tsconfig.json`
Currently, the guides have these instructions:

```
touch tsconfig.json
```

And spell out the contents of it like so:

```
{
  "compilerOptions": {
    "sourceMap": true,
    "outDir": "dist",
    "strict": true,
    "lib": ["esnext"],
    "esModuleInterop": true
  }
}
```

An instruction to set up TypeScript using the proper TypeScript CLI seems more appropriate:

```
npx tsc init
```

### PR DESCRIPTION
**Pull request by ObamaTheLlama114**
Changes:
- Changed creating tsconfig file to running tcs init in getting started
- Changed autoincrement to sequence in cockroachdb in getting started
- Removed redundant mongodb connection strings in mongodb getting started
- Removed comment model and added address type in mongodb getting started

Related issues:
- #3306
- #2916
- #3340
- #3769

### CODE REVIEW

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx
Comment (janpio) [inline content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx:538]: Why unrelated changes to MongoDB?

**Review by nikolasburk — APPROVED** [review]
Looks great!!

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx
Comment (janpio) [inline content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx:540]: Does this change take the follow up differences in generated SQL or similar (if the getting started has this) into account?

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database.mdx
Comment (janpio) [inline content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database.mdx:47]: Why remove this Atlas specific example?

File: content/100-getting-started/01-quickstart.mdx
Comment (keerlu) [inline content/100-getting-started/01-quickstart.mdx]: Does this work for you? When I run it after following the steps immediately above I get the error 

```
error TS6231: Could not resolve the path 'init' with the extensions: '.ts', '.tsx', '.d.ts', '.cts', '.d.cts', '.mts', '.d.mts'.
  The file is in the program because:
    Root file specified for compilation
```

**Comment by keerlu** [comment]
Looks good from a writing point of view (so just need the comments above addressed)

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database.mdx
Comment (meletj) [reply content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database.mdx:47]: Actually I think that was a confusion in the issue. The link to Atlas should definitely stay

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx
Comment (ObamaTheLlama114) [reply content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx:540]: Yes. Generated sql uses sequence instead of autoincrement.

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database.mdx
Comment (ObamaTheLlama114) [reply content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database.mdx:47]: The link is the same link as there is a little further up in the page, and none of the test provides any new information, so it seemed like there is no point in having it there.

File: content/100-getting-started/01-quickstart.mdx
Comment (ObamaTheLlama114) [reply content/100-getting-started/01-quickstart.mdx]: This should have been tsc --init. Will fix

Comment (ObamaTheLlama114) [reply content/100-getting-started/01-quickstart.mdx]: fixed

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database.mdx
Comment (meletj) [reply content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database.mdx:47]: @janpio before I merge this one, wanted to check if William answered your question or if you had any other feedback.

Comment (janpio) [reply content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database.mdx:47]: Yes. I think in other guides we use a more generic URL for the first one, so I assumed that would be the case here as well.

**Pull request merged**
